pkgs/servers/sql/postgresql/default.nix

```diff
@@ -58,10 +58,10 @@ let
         substituteInPlace "$out/lib/pgxs/src/Makefile.global" --replace ${stdenv.cc}/bin/ld ld
       '';
 
-    postFixup =
+    postFixup = lib.optionalString (!stdenv.isDarwin)
       ''
         # initdb needs access to "locale" command from glibc.
-        wrapProgram $out/bin/initdb --prefix PATH ":" ${glibc.bin}/bin
+        wrapProgram $out/bin/initdb --prefix PATH ":" ${pkgs.glibc.bin}/bin
```

Comment:
Is `pkgs.glibc` neccesary, I think it's fine to use `glibc` directly as long as it's not evaluated on darwin.